File: Languages/Python/params.py
from flask import Flask, request

app = Flask(__name__)

# Query params
@app.route("/contact")
def contact():
	request_params = request.args
	name = request_params.get("name")
	age = request_params.get("age")

# Path params
@app.route("/weather/<city>")
def weather(city):
	return f"<h1>Your city is {city}</h1>"

if __name__ == '__main__':
    app.run(debug=True)

File: Languages/Go/params.go
package main

import (
	"fmt"
	"net/http"
)

// Query Params
func contact(writer http.ResponseWriter, request *http.Request) {
	values := request.URL.Query()
	fmt.Printf("Values: %v", values)
}

// Path Params
func weather(writer http.ResponseWriter, request *http.Request) {
	city := request.PathValue("city")
	fmt.Printf("City: %v", city)
}

func main() {
	// Setup server
	mux := http.NewServeMux()

	// Add routes
	mux.HandleFunc("/contact", contact)
	mux.HandleFunc("/weather/{city}", weather)

	// Run
	http.ListenAndServe(":8090", mux)
}
